source/operations/install-deploy-manage/upgrade-minio-operator.rst

```diff
@@ -36,15 +39,147 @@ The following table lists the upgrade paths from previous versions of the MinIO
 
 .. _minio-k8s-upgrade-minio-operator-procedure:
 
-Upgrade MinIO Operator 4.2.3 and Later to |operator-version-stable|
+Upgrade MinIO Operator 4.5.8 and Later to |operator-version-stable|
 -------------------------------------------------------------------
 
 .. admonition:: Prerequisites
    :class: note
 
    This procedure requires the following:
 
-   - You have an existing MinIO Operator deployment running 4.2.3 or later
+   - You have an existing MinIO Operator deployment running 4.5.8 or later
+   - Your Kubernetes cluster runs 1.19.0 or later
+   - Your local host has ``kubectl`` installed and configured with access to the Kubernetes cluster
+
+This procedure upgrades the MinIO Operator from any 4.5.8 or later release to |operator-version-stable|.
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+
+The following changes apply for Operator v5.0.0 or later:
+
+- The Operator's Graphical User Interface now comes bundled with the same container as the Operator.
+- The ``.spec.s3`` field is replaced by the ``.spec.features`` field.
+- The ``.spec.credsSecret`` field is replaced by the ``.spec.configuration`` field.
+
+  This secret should hold all the environment variables for the MinIO deployment that contain sensitive information and should not show in ``.spec.env``.
```

Comment:
Is `this secret` referring to `.spec.credsSecret`? Perhaps something like `.spec.credsSecret contains sensitive environment variables that should not be shown in .spec.env."

Reply:
I went ahead and swapped "This secret" for the actual field name instead. Also added your suggestion, @ravindk89.